alot/commands/thread.py

```diff
@@ -648,7 +654,7 @@ class PipeCommand(Command):
     repeatable = True
 
     def __init__(self, cmd, all=False, separately=False, background=False,
-                 shell=False, notify_stdout=False, format='raw',
+                 shell=False, notify_stdout=False, format='raw', as_file=False,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'format'](https://app.codacy.com/manual/patricktotzke/alot/pullRequest?prid=5220757)